### PR TITLE
update versions

### PR DESCRIPTION
**Pull request by aaabramov**
Scala:
- 2.12.[8,11] -> 2.12.12
- 2.13.[2.3] -> 2.13.4

Sbt: 
- 1.3.13 -> 1.4.6

sbt-giter8:
- 0.11.0 -> 0.13.1

sbt-paradox:
- 0.3.5 -> 0.4.2

scalatest:
- 3.0.8 -> 3.2.3

### CODE REVIEW

File: src/main/g8/build.sbt [hunk 1]
```diff
@@ -17,6 +17,6 @@ lazy val root = (project in file(".")).
 
       "com.typesafe.akka" %% "akka-http-testkit"        % akkaHttpVersion % Test,
       "com.typesafe.akka" %% "akka-actor-testkit-typed" % akkaVersion     % Test,
-      "org.scalatest"     %% "scalatest"                % "3.0.8"         % Test
+      "org.scalatest"     %% "scalatest"                % "3.2.3"         % Test
```
Comment (raboof) [inline src/main/g8/build.sbt:20]: We should definitely update this to 3.1, but I'm not sure about 3.2: Akka still depends on 3.1 (https://github.com/akka/akka/issues/29368) and letting this diverge might cause trouble.

**Review by raboof — COMMENTED** [review]
Great, thanks a lot! I'm not sure about scalatest, otherwise LGTM

File: src/main/g8/build.sbt (hunk 1 above)
Comment (aaabramov) [reply src/main/g8/build.sbt:20]: Ok, will decrease to 3.1.X, moment

Comment (aaabramov) [reply src/main/g8/build.sbt]: Downgraded to 3.1.4

**Review by raboof — APPROVED** [review]

**Comment by raboof** [comment]
thanks!

**Pull request merged**